test/db/archos/linux-x64/dbg_step_back

```diff
@@ -29,25 +32,35 @@ db @ 0x0040052f
 dc
 dts+
 dc
-dr rax,rbx,rcx,r10,rbp,rip
+dr rax
+dr rbx
+dr rcx
+dr r10
+dr rbp
+dr rip
 dsb
 dsb
-dr rax,rbx,rcx,r10,rbp,rip
+dr rax
+dr rbx
+dr rcx
+dr r10
+dr rbp
+dr rip
```

Comment:
I was thinking, why not allowing dr/ar to have multiple args as array of values?
so this would become `dr rax rbx rcx r10 rbp rip`

Reply:
if the problem is when you assign, the problem is non-existent because the arg will still be one 
`dr rax=0x1243 rbx=0x1243 rcx=0x1243 r10=0x1243 rbp=0x1243 rip=0x1243`

Reply:
We would lose the ability to write `dr rax = 0x1234`, but at the same time it would also be more in line with `e`. Maybe I will refactor out the code from there and reuse it in both, which would also address @XVilka's suggestion from above.

Reply:
Sounds good to me, @ret2libc thoughts?

Reply:
In fact the pattern is general enough, that the parsing of such cases can be generalized and shared between those, and maybe somewhere else in the future.

Reply:
ok for me too